<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_main</class>
 <widget class="QMainWindow" name="frm_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #2c2c2c;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="btn_cliente">
               <property name="styleSheet">
                <string notr="true">QPushButton{
    background-color: #f5f5f5;
    border: 2px solid #cccccc;
    border-radius: 10px; 
    padding: 10px; 
    color: #333333; 
    font-size: 14px;
	background-image: url(:/icon_cliente/cliente.png);
    background-repeat: no-repeat;
    background-position: center;
	transition: all 0.3s ease;
}
QPushButton:hover{
    background-color: #e0e0e0;
    border-color: #aaaaaa;
	transform: scale(1.05); 
}
QPushButton:pressed{
	background-color: #d6d6d6;
	border-color: #888888;
	transform: scale(0.95);
}</string>
               </property>
               <property name="text">
                <string>ㅤㅤ
ㅤㅤㅤㅤㅤㅤㅤ</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_7" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="btn_fornecedor">
               <property name="styleSheet">
                <string notr="true">QPushButton{
    background-color: #f5f5f5;
    border: 2px solid #cccccc;
    border-radius: 10px; 
    padding: 10px; 
    color: #333333; 
    font-size: 14px;
	background-image: url(:/icon_fornecedor/fornecedor.png);
    background-repeat: no-repeat;
    background-position: center;
	transition: all 0.3s ease;
}
QPushButton:hover{
    background-color: #e0e0e0;
    border-color: #aaaaaa;
	transform: scale(1.05); 
}
QPushButton:pressed{
	background-color: #d6d6d6;
	border-color: #888888;
	transform: scale(0.95);
}</string>
               </property>
               <property name="text">
                <string>ㅤㅤ
ㅤㅤㅤㅤㅤㅤㅤ</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="btn_produtos">
               <property name="styleSheet">
                <string notr="true">QPushButton{
    background-color: #f5f5f5;
    border: 2px solid #cccccc;
    border-radius: 10px; 
    padding: 10px; 
    color: #333333; 
    font-size: 14px;
	background-image: url(:/icon_produtos/produtos.png);
    background-repeat: no-repeat;
    background-position: center;
	transition: all 0.3s ease;
}
QPushButton:hover{
    background-color: #e0e0e0;
    border-color: #aaaaaa;
	transform: scale(1.05); 
}
QPushButton:pressed{
	background-color: #d6d6d6;
	border-color: #888888;
	transform: scale(0.95);
}</string>
               </property>
               <property name="text">
                <string>ㅤㅤ
ㅤㅤㅤㅤㅤㅤㅤ</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QPushButton" name="btn_tarefas">
               <property name="styleSheet">
                <string notr="true">QPushButton{
    background-color: #f5f5f5;
    border: 2px solid #cccccc;
    border-radius: 10px; 
    padding: 10px; 
    color: #333333; 
    font-size: 14px;
	background-image: url(:/icon_tarefas/tarefa.png);
    background-repeat: no-repeat;
    background-position: center;
	transition: all 0.3s ease;
}
QPushButton:hover{
    background-color: #e0e0e0;
    border-color: #aaaaaa;
	transform: scale(1.05); 
}
QPushButton:pressed{
	background-color: #d6d6d6;
	border-color: #888888;
	transform: scale(0.95);
}</string>
               </property>
               <property name="text">
                <string>ㅤㅤ
ㅤㅤㅤㅤㅤㅤㅤ</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_14" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_11"/>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="btn_vendas">
               <property name="styleSheet">
                <string notr="true">QPushButton{
    background-color: #f5f5f5;
    border: 2px solid #cccccc;
    border-radius: 10px; 
    padding: 10px; 
    color: #333333; 
    font-size: 14px;
	background-image: url(:/icon_vendas/vendas.png);
    background-repeat: no-repeat;
    background-position: center;
	transition: all 0.3s ease;
}
QPushButton:hover{
    background-color: #e0e0e0;
    border-color: #aaaaaa;
	transform: scale(1.05); 
}
QPushButton:pressed{
	background-color: #d6d6d6;
	border-color: #888888;
	transform: scale(0.95);
}</string>
               </property>
               <property name="text">
                <string>ㅤㅤ
ㅤㅤㅤㅤㅤㅤㅤ</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QWidget" name="widget_12" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QPushButton" name="btn_pagar">
               <property name="styleSheet">
                <string notr="true">QPushButton{
    background-color: #f5f5f5;
    border: 2px solid #cccccc;
    border-radius: 10px; 
    padding: 10px; 
    color: #333333; 
    font-size: 14px;
	background-image: url(:/icon_pagar/pagar.png);
    background-repeat: no-repeat;
    background-position: center;
	transition: all 0.3s ease;
}
QPushButton:hover{
    background-color: #e0e0e0;
    border-color: #aaaaaa;
	transform: scale(1.05); 
}
QPushButton:pressed{
	background-color: #d6d6d6;
	border-color: #888888;
	transform: scale(0.95);
}</string>
               </property>
               <property name="text">
                <string>ㅤㅤ
ㅤㅤㅤㅤㅤㅤㅤ</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_10" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QPushButton" name="btn_receber">
               <property name="styleSheet">
                <string notr="true">QPushButton{
    background-color: #f5f5f5;
    border: 2px solid #cccccc;
    border-radius: 10px; 
    padding: 10px; 
    color: #333333; 
    font-size: 14px;
	background-image: url(:/icon_receber/receber.png);
    background-repeat: no-repeat;
    background-position: center;
	transition: all 0.3s ease;
}
QPushButton:hover{
    background-color: #e0e0e0;
    border-color: #aaaaaa;
	transform: scale(1.05); 
}
QPushButton:pressed{
	background-color: #d6d6d6;
	border-color: #888888;
	transform: scale(0.95);
}</string>
               </property>
               <property name="text">
                <string>ㅤㅤ
ㅤㅤㅤㅤㅤㅤㅤ</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_11" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QPushButton" name="btn_bancos">
               <property name="styleSheet">
                <string notr="true">QPushButton{
    background-color: #f5f5f5;
    border: 2px solid #cccccc;
    border-radius: 10px; 
    padding: 10px; 
    color: #333333; 
    font-size: 14px;
	background-image: url(:/icon_bancos/bancos.png);
    background-repeat: no-repeat;
    background-position: center;
	transition: all 0.3s ease;
}
QPushButton:hover{
    background-color: #e0e0e0;
    border-color: #aaaaaa;
	transform: scale(1.05); 
}
QPushButton:pressed{
	background-color: #d6d6d6;
	border-color: #888888;
	transform: scale(0.95);
}</string>
               </property>
               <property name="text">
                <string>ㅤㅤ
ㅤㅤㅤㅤㅤㅤㅤ</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">QLabel{
	background: url(:/icon_logoInicial/AvsLogo.png);
	background-repeat: no-repeat; 
    background-position: center; 
}</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤ&lt;/p&gt;&lt;p&gt;ㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤ&lt;/p&gt;&lt;p&gt;ㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>975</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
    background-color: #2c2c2c;
    color: #ffffff;
    font: 12pt &quot;Segoe UI&quot;;
}

QMenuBar::item {
    background: transparent;
    padding: 4px 12px;
    margin: 2px;
    border-radius: 6px;
}

QMenuBar::item:selected { 
    background: #444444; 
}

QMenuBar::item:pressed {
    background: #555555;
}

QMenu {
    background-color: #2c2c2c;
    color: #ffffff;
    border: 1px solid #444444;
    padding: 6px;
}

QMenu::item {
    background: transparent;
    padding: 6px 20px;
    border-radius: 4px;
}

QMenu::item:selected {
    background: #444444;
}
</string>
   </property>
   <widget class="QMenu" name="menuCadastro">
    <property name="title">
     <string>Cadastro</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionCliente"/>
    <addaction name="actionFornecedor"/>
    <addaction name="actionProdutos"/>
    <addaction name="actionTarefas"/>
    <addaction name="actionFuncion_rio"/>
   </widget>
   <widget class="QMenu" name="menuFinanceiro">
    <property name="title">
     <string>Financeiro</string>
    </property>
    <addaction name="actionBanco"/>
    <addaction name="actionContas_pagar"/>
    <addaction name="actionContas_receber"/>
   </widget>
   <widget class="QMenu" name="menuPDV">
    <property name="title">
     <string>PDV</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
   </widget>
   <addaction name="menuCadastro"/>
   <addaction name="menuFinanceiro"/>
   <addaction name="menuPDV"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar {
    background-color: #2c2c2c;
    color: #ffffff;
    font: 11pt &quot;Segoe UI&quot;;
    border-top: 1px solid #444444;
}

QStatusBar::item {
    border: none;
    background: transparent;
    padding: 2px 6px;
}
</string>
   </property>
  </widget>
  <action name="actionCliente">
   <property name="text">
    <string>Cliente</string>
   </property>
  </action>
  <action name="actionFornecedor">
   <property name="text">
    <string>Fornecedor</string>
   </property>
  </action>
  <action name="actionProdutos">
   <property name="text">
    <string>Produtos</string>
   </property>
  </action>
  <action name="actionTarefas">
   <property name="text">
    <string>Tarefas</string>
   </property>
  </action>
  <action name="actionBanco">
   <property name="text">
    <string>Banco</string>
   </property>
  </action>
  <action name="actionContas_pagar">
   <property name="text">
    <string>Contas a pagar</string>
   </property>
  </action>
  <action name="actionContas_receber">
   <property name="text">
    <string>Contas a receber</string>
   </property>
  </action>
  <action name="actionFuncion_rio">
   <property name="text">
    <string>Funcionário</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Img/icon_cliente.qrc"/>
  <include location="../Img/icon_bancos.qrc"/>
  <include location="../Img/icon_sobre.qrc"/>
  <include location="../Img/icon_vendas.qrc"/>
  <include location="../Img/icon_receber.qrc"/>
  <include location="../Img/icon_pagar.qrc"/>
  <include location="../Img/icon_tarefas.qrc"/>
  <include location="../Img/icon_produtos.qrc"/>
  <include location="../Img/icon_fornecedor.qrc"/>
  <include location="../Img/icon_logoInicial.qrc"/>
 </resources>
 <connections/>
</ui>
